edgexfoundry/edgex-go
# feat(metadata): implement POST device resource

Signed-off-by: Ginny Guan <ginny@iotechsys.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have opened a PR for the related docs change (if not, why?) update core-metadata Swagger file in this PR
  <link to docs PR>

## Testing Instructions

1. run core-metadata from this branch 
2. POST /deviceprofile/resource and verify the responses

## New Dependency Instructions (If applicable)
